### ISSUE TITLE
Could not find a sequence in header2rseq hash table

### ISSUE THREAD

**Issue by aoharrison** [mooreryan/group_proteins#10]
I got the error running v0.0.5

Command:

```
time /home/moorer/software/pasv/bin/pasv -a clustalo -p '--threads 1' -s -1 -e -1 -o test.pep -t 1 -r Class3_refs.fasta -d aln_dir_test2.pep -q test.pep 79 290 543 546 561 564 580
```

Error:

```
FATAL -- src/pasv.c:109 -- Could not find seq 'NODE_24_length_18210_cov_279.773_18060_18210_19 size=50.3333333333333 gc=0.410596026490066 start=18060 stop=18210 strand=+ frame=0 model=phage score=28.4972 type=lack_stop cal' in the header2rseq hash table
```

[test.pep.txt](https://github.com/mooreryan/pasv/files/1166343/test.pep.txt)

**Comment by aoharrison** [comment]
Here is the reference file I used when I got the above error.

[Class3_refs.txt](https://github.com/mooreryan/pasv/files/1166460/Class3_refs.txt)

**Comment by mooreryan** [comment]
The problem was that aligners were cutting off long header names. Now sequences are keyed off of the ID rather than the full header.

This commit should fix the problem c7599b817fe1f696ac03e3cc451efbe1ee4286db.
